src/main/fc/config.c

```diff
@@ -156,7 +156,15 @@ PG_RESET_TEMPLATE(systemConfig_t, systemConfig,
 #endif
 
 #ifdef BEEPER
-PG_REGISTER(beeperConfig_t, beeperConfig, PG_BEEPER_CONFIG, 0);
+PG_REGISTER_WITH_RESET_FN(beeperConfig_t, beeperConfig, PG_BEEPER_CONFIG, 0);
```

Comment:
Since this is simply setting a member variable to true, a reset template rather than a reset function should be used.